scripts/etc/init.d/rcS

```diff
@@ -301,40 +301,43 @@ esac
 # configure different kinds of presets
 if [ "$cdebootstrap_cmdline" = "" ]; then
 
-    # minimal
-    cdebootstrap_cmdline="--flavour=minimal --include=kmod,fake-hwclock,ifupdown,net-tools,ntp,openssh-server$dhcp_packages"
+    # from small to large: base, minimal, server
+    # not very logical that minimal > base, but that's how it was historically defined
 
-    # add dhcp client if using dhcp
-    if [ "$ip_addr" = "dhcp" ]; then
-        cdebootstrap_cmdline="$cdebootstrap_cmdline,isc-dhcp-client"
-    fi
+    # base
+    base_packages="${kernel_meta_package},cpufrequtils,kmod"
 
-    # add latest kernel if rootfstype requires a module. 
-    # Changed it so it is always installed.
-    # Firmware can't be installed just yet, we'll do so at the end
-    if [ "$kernel_module" = true ] ; then
-        cdebootstrap_cmdline="$cdebootstrap_cmdline,${kernel_meta_package}"
-    fi
+    # minimal
+    minimal_packages="fake-hwclock,ifupdown,net-tools,ntp,openssh-server"
```

Comment:
I had it like that at first, but changed it into what it is now. Don't recall right now what the reasoning of it was ...